File: wavesplatform/src/main/java/com/wavesplatform/sdk/model/response/TransactionsInfo.java
package com.wavesplatform.sdk.model.response;

/**
 * Created by anonymous on 11.07.17.
 */

public class TransactionsInfo {
    public int type;
    public String id;
    public String sender;
    public String senderPublicKey;
    public Long fee;
    public String timestamp;
    public String signature;
    public String assetId;
    public String name;
    public String description;
    public Long quantity;
    public int decimals;
    public boolean reissuable;
    public Long height;

    public TransactionsInfo(String assetId, String name, int decimals) {
        this.assetId = assetId;
        this.name = name;
        this.decimals = decimals;
    }
}

File: wavesplatform/src/main/java/com/wavesplatform/sdk/net/model/response/TransactionsInfo.kt
/*
 * Created by Eduard Zaydel on 1/4/2019
 * Copyright © 2019 Waves Platform. All rights reserved.
 */

package com.wavesplatform.sdk.net.model.response

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class TransactionsInfo(var assetId: String, var name: String, var decimals: Int) : Parcelable {
    var type: Int = 0
    var id: String? = null
    var sender: String? = null
    var senderPublicKey: String? = null
    var fee: Long? = null
    var timestamp: String? = null
    var signature: String? = null
    var description: String? = null
    var quantity: Long? = null
    var reissuable: Boolean = false
    var height: Long? = null
}
